admin/services/check-start.service/check-start.service.js

```diff
@@ -17,14 +17,17 @@ const checkFormService = require('../check-form.service')
 const configService = require('../config.service')
 const dateService = require('../date.service')
 const pinGenerationDataService = require('../data-access/pin-generation.data.service')
-const pinGenerationService = require('../pin-generation.service')
+const pinTimestampService = require('../pin-timestamp.service')
 const prepareCheckService = require('../prepare-check.service')
 const queueNameService = require('../queue-name-service')
 const sasTokenService = require('../sas-token.service')
 const redisCacheService = require('../data-access/redis-cache.service')
 const redisKeyService = require('../redis-key.service')
 const oneMonthInSeconds = 2592000
 
+const fourPmToday = moment().startOf('day').add(16, 'hours')
```

Comment:
This is an expression that will resolve to a value, marked as const.  How does it get updated for the next day?